pom.xml

```diff
@@ -373,7 +373,7 @@
         <version.org.reactivestreams>1.0.2</version.org.reactivestreams>
         <version.org.syslog4j>0.9.30</version.org.syslog4j>
         <version.org.wildfly.arquillian>2.1.1.Final</version.org.wildfly.arquillian>
-        <version.org.wildfly.core>6.0.0.Alpha2</version.org.wildfly.core>
+        <version.org.wildfly.core>6.0.0.Alpha3</version.org.wildfly.core>
```

Comment:
This change should be removed (the WildFly Core version will be updated via https://github.com/wildfly/wildfly/pull/11396).

Reply:
Fixed